modular_skyrat/modules/ashwalkers/code/effects/ash_ritual.dm

```diff
@@ -0,0 +1,211 @@
+	if(east_ritual_component)
+		var/east_check = locate(east_ritual_component) in get_step(east_rune, EAST)
+		if(!east_check)
+			ritual_fail(east_rune)
+			return FALSE
+		if(consume_east_component)
+			qdel(east_check)
+			east_rune.balloon_alert_to_viewers("the east component has been consumed...")
+		new ritual_effect(east_rune.loc)
+		sleep(ritual_time)
+		return TRUE
+	return TRUE
+
+/datum/ash_ritual/proc/check_west(obj/effect/ash_rune/west_rune)
+	if(west_ritual_component)
+		var/west_check = locate(west_ritual_component) in get_step(west_rune, WEST)
+		if(!west_check)
+			ritual_fail(west_rune)
+			return FALSE
+		if(consume_west_component)
+			qdel(west_check)
+			west_rune.balloon_alert_to_viewers("the west component has been consumed...")
+		new ritual_effect(west_rune.loc)
+		sleep(ritual_time)
+		return TRUE
+	return TRUE
+
+/datum/ash_ritual/proc/ritual_fail(obj/effect/ash_rune/failed_rune)
+	new ritual_effect(failed_rune.loc)
+	failed_rune.balloon_alert_to_viewers("the ritual has failed...")
+	failed_rune.current_ritual = null
+	in_use = FALSE
+	return
+
+/datum/ash_ritual/proc/ritual_success(obj/effect/ash_rune/success_rune)
+	new ritual_effect(success_rune.loc)
+	success_rune.balloon_alert_to_viewers("the ritual has been successful...")
+	if(ritual_success_item)
+		new ritual_success_item(get_turf(success_rune))
+	success_rune.current_ritual = null
+	in_use = FALSE
+	return TRUE
+
+/datum/ash_ritual/summon_staff
+	name = "Summon Ash Staff"
+	north_ritual_component = /obj/item/stack/sheet/mineral/wood
+	south_ritual_component = /obj/item/organ/regenerative_core
+	ritual_success_item = /obj/item/ash_staff
+
+/datum/ash_ritual/summon_necklace
+	name = "Summon Draconic Necklace"
+	north_ritual_component = /obj/item/stack/sheet/bone
+	south_ritual_component = /obj/item/organ/regenerative_core
+	east_ritual_component = /obj/item/stack/sheet/sinew
+	west_ritual_component = /obj/item/stack/sheet/sinew
+	ritual_success_item = /obj/item/clothing/neck/necklace/ashwalker
+
+/datum/ash_ritual/summon_key
+	name = "Summon Skeleton Key"
+	north_ritual_component = /obj/item/stack/sheet/bone
+	south_ritual_component = /obj/item/stack/sheet/bone
+	east_ritual_component = /obj/item/stack/sheet/bone
+	west_ritual_component = /obj/item/stack/sheet/bone
+	ritual_success_item = /obj/item/skeleton_key
+
+/datum/ash_ritual/summon_cursed_knife
+	name = "Summon Cursed Ash Knife"
+	north_ritual_component = /obj/item/organ/regenerative_core
+	south_ritual_component = /obj/item/forging/reagent_weapon/dagger
+	east_ritual_component = /obj/item/stack/sheet/bone
+	west_ritual_component = /obj/item/stack/sheet/sinew
+	ritual_success_item = /obj/item/cursed_dagger
+
+/datum/ash_ritual/summon_tendril_seed
+	name = "Summon Tendril Seed"
+	north_ritual_component = /obj/item/organ/regenerative_core
+	south_ritual_component = /obj/item/cursed_dagger
+	east_ritual_component = /obj/item/crusher_trophy/goliath_tentacle
```

Comment:
currently ashies don't get crusher trophies from regular fauna, just megafauna (i'll look into figuring out a good way of doing fauna trophies later)

Reply:
that would be good... maybe a ritual to get the trophies? lmao... just kidding-- unless?